warehouse/ingest-core/src/main/java/datawave/iterators/PropogatingIterator.java

```diff
@@ -319,11 +331,49 @@ public boolean validateOptions(Map<String,String> options) {
         
         PropogatingCombiner propAgg = null;
         
+        Map<String,Map<String,String>> allAggOptions = new HashedMap();
+        // make a first pass of the options to find any aggregator options and build them up for each aggregator
         for (Entry<String,String> familyOption : options.entrySet()) {
+            String option = familyOption.getKey();
+            // if this property specifies aggregator options skip
+            if (option.contains(AGGREGATOR_OPTS)) {
+                // get the aggregator name
+                String[] splits = option.split("\\.");
+                String aggName = splits[0];
+                if (splits.length < 3) {
+                    throw new IllegalArgumentException("When specifying options for an aggregator specify <cf>." + AGGREGATOR_OPTS + "<option>");
+                }
+                
+                Map<String,String> aggOptions = allAggOptions.get(aggName);
+                if (aggOptions == null) {
+                    aggOptions = new HashMap<>();
+                    allAggOptions.put(aggName, aggOptions);
+                }
+                
+                // strip off the first two sections of the option to build an option for the aggregator
+                String aggOption = option.replace(splits[0] + "." + splits[1] + ".", "");
```

Comment:
if we've already split the option into it's three components (e.g. COLFAM.opts.MYOPTION -> 1) COLFAM, 2) opts, 3) MYOPTION) why can't jkust set aggOption as follows?  `aggOption = splits[2]`